### ISSUE TITLE
Signoff checklist: Does FPGA_TIMING actually make sense?

### ISSUE THREAD

**Issue by rswarbrick** [lowRISC/opentitan#6935]
The D2 signoff checklist contains an FPGA_TIMING item that says "FPGA synthesis timing meet (Fmax-10%) target or better". What does this mean in the context of OpenTitan? If it's just that the FPGA syntheses that we do in CI hit their timing, then this is guaranteed by CI.

Maybe the item can be removed?

@eunchan (I think you wrote the original text?)
@imphil (as resident CI guru!)
@tomroberts-lowrisc (as lowRISC HW lead)

**Comment by imphil** [comment]
> If it's just that the FPGA syntheses that we do in CI hit their timing, then this is guaranteed by CI.

Yes, that's what is meant by that. (When we wrote the checklist it wasn't clear that CI would check FPGA timing, and actually it didn't.)

But even with CI checking the FPGA timing, it's good to have it in the list because it motivates having the check in CI in the first place.

I'd therefore just leave things as-is and call it an easy item to check off in the list.

**Comment by rswarbrick** [comment]
Hmm, but there are lots of things in CI that aren't motivated by this checklist. Also, it's a bit awkward that "Fmax" isn't really defined. Especially, since the text sounds like there's some wiggle room and there really isn't with the CI check.

Maybe we should reword it to something like "Block is synthesized for FPGA in CI and meets timing."?

**Comment by imphil** [comment]
The suggested reword sounds fine to me.
